### PR TITLE
Implement translations in the views

### PR DESCRIPTION
**Pull request by Kilian**
This is a monster PR, sorry about that :) Should have all views translated and is rebased on latest master to should merge cleanly. Best to get that done asap to prevent conflicts with other work-in-progress!

### CODE REVIEW

**Comment by fossecode** [comment]
Fantastic work! Will look through it now

File: app/locales/en.json [hunk 1]
```diff
@@ -7,5 +7,170 @@
...
+  "Save the following link somewhere, or bookmark it in your browser.": "Save the following link somewhere, or bookmark it in your browser.",
+  "That way you can update your report if your health condition changes.": "That way you can update your report if your health condition changes.",
+  "I've noted down the link.": "I've noted down the link.",
+  "Proceed": "Proceed",
+  "The link has been copied": "The link has been copied.",
+  "Could not copy the link, please copy it yourself": "Could not copy the link, please copy it yourself",
+  "Contributors": "Contributors",
+  "This website is made with help from volunteers. In no particular order:": "This website is made with help from volunteers. In no particular order:",
+  "...and others": "...and others",
+  "The code for the website can be found": "Koden for nettsiden finnes",
```
Comment (fossecode) [inline app/locales/en.json]: Should be English text here, not Norwegian

File: app/locales/en.json [hunk 2]
```diff
@@ -7,5 +7,170 @@
...
+  "The spread of COVID-19 in our country is unknown. Help us create a better overview.": "The spread of COVID-19 in our country is unknown. Help us create a better overview.",
+  "Contribute anonymously": "Contribute anonymously",
+  "Report your health condition anonymously": "Report your health condition anonymously",
+  "Insight for everyone": "Insight for everyone",
+  "The data is open to everyone. Look at maps and graphs, or download the data and create your own visualizations.": "The data is open to everyone. Look at maps and graphs, or download the data and create your own visualizations.",
+  "Everything is open": "Everything is open",
+  "The website is open source and built by volunteers. Feel free to join.": "The website is open source and built by volunteers. Feel free to join.",
+  "Total reports": "Total reports",
+  "Infected / Have been tested": "Infected / Have been tested",
+  "In close contact with someone who's infected": "IIn close contact with someone who's infected",
```
Comment (fossecode) [inline app/locales/en.json]: `IIn` -> `In`

File: app/locales/en.json [hunk 3]
```diff
@@ -7,5 +7,170 @@
...
+  "Has symptoms": "Has symptoms",
+  "You must approve the privacy statement to report your data.": "You must approve the privacy statement to report your data.",
+  "Your state of health": "Your state of health",
+  "Report your health condition": "Report your health condition",
+  "Thank you for your contribution! Use this link if your health condition changes:": "Thank you for your contribution! Use this link if your health condition changes:",
+  "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ": "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ",
+  "Geography and demography": "Geography and demography",
+  "/healthcondition/": "/healthcondition/",
```
Comment (fossecode) [inline app/locales/en.json]: I don't think we need this after https://github.com/BustByte/coronastatus/pull/199

File: app/locales/en.json (hunk 1 above)
Comment (Kilian) [reply app/locales/en.json]: fixing!

File: app/locales/en.json (hunk 2 above)
Comment (Kilian) [reply app/locales/en.json]: fixing!

File: app/locales/en.json (hunk 3 above)
Comment (Kilian) [reply app/locales/en.json]: yep, I made sure the templates use the variables. Removed!

File: app/locales/en.json [hunk 4]
```diff
@@ -7,5 +7,170 @@
...
+  "Your age": "Your age",
+  "Choose your age": "Choose your age",
+  "Biological gender": "Biological gender/Sex",
```
Comment (fossecode) [inline app/locales/en.json:73]: Should probably not use `gender` at all?

Comment (Kilian) [reply app/locales/en.json:73]: I don't have a strong opinion on this other than having "biological gender" there making it easier to understand "sex" in context.

File: app/locales/en.json [hunk 5]
```diff
@@ -7,5 +7,170 @@
...
+  "When did the symptoms start?": "When did the symptoms start?",
+  "How will my data be used?": "How will my data be used?",
+  "Your health information is stored anonymously, and can not be used to identify you.": "Your health information is stored anonymously, and can not be used to identify you.",
+  "Our privacy policy can be found here": "Our privacy policy can be found here",
+  "I agree to my being data in accordance with the privacy statement": "I agree to my being data in accordance with the privacy statement",
+  "Submit report": "Submit report",
+  "Statistics": "Statistics",
+  "Download the dataset as CSV": "Download the dataset as CSV",
+  "COVID-19 infected": "COVID-19 infected",
+  "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.": "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.",
```
Comment (fossecode) [inline app/locales/en.json]: Did you check if using `<%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %>`works? Should maybe rather be split up.

File: app/locales/en.json [hunk 6]
```diff
@@ -7,5 +7,170 @@
...
+  "Everyone who has submitted a report, also has:": "Everyone who has submitted a report, also has:",
+  "Total number of reports": "Total number of reports",
+  "Positive test results and close contact": "Positive test results and close contact",
+  "Clarifications data basis": "Clarifications data basis",
+  "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:": "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:",
+  "Symptom": "Symptom",
+  "Added to form": "Added to form",
+  "Headache": "Headache",
+  "Sore throat": "Sore throat",
+  "Loss of sense of taste": "Loss of sense of taste",
```
Comment (fossecode) [inline app/locales/en.json]: There are some dupes, don't know if that matters or not.

File: app/locales/en.json [hunk 7]
```diff
@@ -7,5 +7,170 @@
...
+  "Loss of sense of smell": "Loss of sense of smell",
+  "Mucus cough": "Mucus cough",
+  "Stuffy or runny nose": "Stuffy or runny nose",
+  "Statistics": "Statistieken",
```
Comment (fossecode) [inline app/locales/en.json]: Line 135-172 contains NL translations, should be removed

File: app/locales/en.json (hunk 6 above)
Comment (Kilian) [reply app/locales/en.json]: That's what happens in merge-hell :( We can remove dupes

File: app/locales/en.json (hunk 5 above)
Comment (Kilian) [reply app/locales/en.json]: Yeah I overlooked this. It's fixed for one instance but I'll (or someone else) go over the others and fix them.

**Comment by Kilian** [comment]
Not all translations made it into the dutch translation file, should be checked.

File: app/views/pages/report.ejs [hunk 8]
```diff
@@ -152,147 +152,152 @@
...
                 </div>
               </div>
             </div>
             <div id="symptom-start-question" class="col-5 col-12-small input-block" style="display: none;">
-              <label class="with-top-margin" for="symptom-start">Når startet symptomene?</label>
+              <label class="with-top-margin" for="symptom-start"><%= __('When did the symptoms start?')%></label>
               <input type="date" name="symptom-start" id="symptom-start" value="<%= symptomStart %>">
             </div>
           </div>
+<<<<<<< HEAD
```
Comment (fossecode) [inline app/views/pages/report.ejs]: Conflict

File: app/views/pages/statistics.ejs [hunk 9]
```diff
@@ -17,133 +16,136 @@
...
             <div class="stat-cell">
-              <h3 class="stat-heading">Symptomer</h3>
+              <h3 class="stat-heading"><%= __('Symptoms') %></h3>
               <canvas id="allSymptomsChart"></canvas>
             </div>
             <div class="stat-cell">
-              <h3 class="stat-heading">Nøkkeltall</h3>
-              <div>Av alle som har rapportert, har:</div>
+              <h3 class="stat-heading"><%= __('Key figures') %></h3>
+              <div><%= __('Everyone who has submitted a report, also has:') %></div>
```
Comment (fossecode) [inline app/views/pages/statistics.ejs:88]: `also have`

File: app/views/pages/report.ejs (hunk 8 above)
Comment (Kilian) [reply app/views/pages/report.ejs]: fixed

**Comment by fossecode** [comment]
👍 as you suggested, we should probably just merge this asap. Me and @michaelmcmillan can test the Norwegian version afterwords, and see if we find any errors.

**Review by fossecode — APPROVED** [review]

**Pull request merged**